geopandas/base.py

```diff
@@ -564,7 +574,15 @@ def simplify(self, *args, **kwargs):
             index=self.index, crs=self.crs)
 
     def relate(self, other):
-        raise NotImplementedError
+        """
+        Returns the DE-9IM intersection matrix for the two geometries
+
+        Parameters
+        ----------
+        other : BaseGeometry or GeoSeries
+            The other geometry to computed the DE-9IM intersection matrix from.
+        """
```

Comment:
I know the other methods currently don't do that yet, but can you add a `Returns` section here? 
It can look something like:

```
Returns
------
Series of strings
```

Reply:
Thank you for your advice!

I added a return section to docstring.
I hope the grammar is correct.